services/cd-service/pkg/service/commit_deployment.go

```diff
@@ -39,24 +39,11 @@ func (s *CommitDeploymentServer) GetCommitDeploymentInfo(ctx context.Context, in
 	commitDeploymentStatus := make(map[string]*api.AppCommitDeploymentStatus, 0)
 	var jsonCommitEventsMetadata []byte
 	var jsonAllEnvironmentsMetadata []byte
-	applicationReleases := make(map[string][]byte, 0)
+	applicationReleases := make(map[string]map[string]uint64, 0)
 	allApplicationReleasesQuery := `
-SELECT
-  deployments.appname,
-  deployments.json
-FROM (
-  SELECT
-    MAX(eslVersion) AS latest,
-    appname
-  FROM
-    "public"."all_deployments"
-  GROUP BY
-    appname) AS latest
-JOIN
-  "public".all_deployments AS deployments
-ON
-  latest.latest=deployments.eslVersion
-  AND latest.appname=deployments.appname;
+SELECT appname, envname, releaseVersion
```

Comment:
please add the query to dd:
span.SetTag("query", allApplicationReleasesQuery)

Reply:
Done